3ds/libdl/PKGBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: Teddy Astie <TSnake@Outlook.fr>
+# Maintainer:
+
+pkgname=3ds-libdl
+pkgver=1.0.0
+pkgrel=1
+pkgdesc='Extensible libdl library for projects that needs it.'
+arch=('any')
+options=(!strip)
+makedepends=('devkitpro-pkgbuild-helpers')
+source=('dlfcn.c' 'dlfcn.h')
+sha256sums=('9c84be0ef4365a3b2ed3948426ecfbdf8116611d63d5ec42d4af8a551c20ef72' '37c16bc645be27cad69904a53973a8862c37890f7327f6b26998794c62527336')
+groups=('3ds-portlibs')
+
+build() {
+  source /opt/devkitpro/3dsvars.sh
+  
+  arm-none-eabi-gcc $CFLAGS -c -o dlfcn.o dlfcn.c
```

Comment:
Should use `$(CC)` here

Reply:
${CC} is currently not defined in 3dsvars.sh, however, ${TOOL_PREFIX} is defined to `arm-none-eabi-` and is actually used to define ${AR}, so to get ${CC}, and can use ${TOOL_PREFIX}gcc.